tests/test_files/test_coordinates/topol_stateB.top

```diff
@@ -97,7 +97,7 @@ Protein  3
 17       19       1        0.14490  282001.6
 19       20       1        0.15260  259408.0
 19       24       1        0.15220  265265.6
-19       27       3        0.10900 400.0 19.0
+19       27       1        0.10900 400.0 19.0
```

Comment:
These parameters seem strange, shouldn't there be only 2? Agree, that it should be type 1 though.

Reply:
I don't actually know how the type 3 ended up in there, because it made the test for merging fail on my end, but you merged your PR before me without failing tests even though the 3 was there according to history.

Reply:
and true, it should be only 2 parameters, those seem wrong

Reply:
3 parameters for Morse (type 3), 2 parameters for harmonic (type 1). For stateB there is no need for Morse

Reply:
so where did the more come from?

Reply:
ohhh, @KRiedmiller I get it now. The tests ran on your PR and then you added https://github.com/graeter-group/kimmdy/pull/465/commits/59a29a1576c4a5fea67e2cce8ff4997eda7c5a5a , which disallows non-harmonic potentials for merging, but didn't trigger the tests again, which would have failed.

Reply:
I'll update repository rules such that just satisfying the linter check is not enough to trigger the green checkmark for merging (can still be manually overwritten, but makes this easier to spot).